src/wp-includes/media.php

```diff
@@ -949,32 +948,30 @@ function wp_get_registered_image_subsizes() {
  *     @type bool   $3 Whether the image is a resized image.
  * }
  */
-function wp_get_attachment_image_src( $attachment_id, $size = 'thumbnail', $icon = false ) {
+function wp_get_attachment_preview_src( $attachment_id, $size = 'thumbnail' ) {
```

Comment:
This is not an issue in this PR, but it would be great if for the eventual core PR you could change the order of the code so that you add the `wp_get_attachment_preview_src()` implementation in a completely different place in this file (probably somewhere closer to the bottom, since it's new) so that it is in a different place. The way it is now makes it challenging to review the PR since it is now completely intertwined with potential changes to `wp_get_attachment_image_src()`, while that now looks like a new function added below - which I know of course that's not the case, but it's just about making it easier to review the individual code changes. Adding the new function in a different place will make this less confusing and thus also less prone to any oversights.